pulsar-websocket/src/main/java/org/apache/pulsar/websocket/AbstractWebSocketHandler.java

```diff
@@ -67,25 +67,29 @@ public abstract class AbstractWebSocketHandler extends WebSocketAdapter implemen
     protected final WebSocketService service;
     protected final HttpServletRequest request;
 
-    protected final TopicName topic;
+    protected TopicName topic;
     protected final Map<String, String> queryParams;
     private static final String PULSAR_AUTH_METHOD_NAME = "X-Pulsar-Auth-Method-Name";
     protected final ObjectReader consumerCommandReader =
             ObjectMapperFactory.getMapper().reader().forType(ConsumerCommand.class);
 
     private ScheduledFuture<?> pingFuture;
 
+    protected String topicsPattern;
```

Comment:
This value is only used for the consumer side. We should move it to the consumer handler.